core/ui/PageTemplate/story.tid

```diff
@@ -1,8 +1,10 @@
 title: $:/core/ui/PageTemplate/story
 tags: $:/tags/PageTemplate
 
+\function tf.story-river.classes() [{$:/info/mediaquery/min-width}match[no]then[tc-story-river]] :else[[tc-story-river tc-primary-container]]
+\function tf.story-river.tabindex() [{$:/info/mediaquery/min-width}match[no]then[]] :else[[-1]]
 \whitespace trim
-<section class="tc-story-river" role="main">
+<section class=<<tf.story-river.classes>> role="main" tabindex=<<tf.story-river.tabindex>>>
```

Comment:
`tf.` function name should only have 1 dot. imo for consistency reasons it should be `tf.story-river-classes`